modules/net-vpc/subnets.tf

```diff
@@ -142,6 +150,41 @@ resource "google_compute_subnetwork" "l7ilb" {
   )
 }
 
+resource "google_compute_subnetwork" "l7rlb" {
+  provider      = google-beta
+  for_each      = local.subnets_l7rlb
+  project       = var.project_id
+  network       = local.network.name
+  region        = each.value.region
+  name          = each.value.name
+  ip_cidr_range = each.value.ip_cidr_range
+  purpose       = "REGIONAL_MANAGED_PROXY"
+  role = (
+    each.value.active || each.value.active == null ? "ACTIVE" : "BACKUP"
+  )
+  description = lookup(
+    local.subnet_descriptions,
+    "${each.value.region}/${each.value.name}",
+    "Terraform-managed."
+  )
+}
+
+resource "google_compute_subnetwork" "psc" {
+  provider      = google-beta
+  for_each      = local.subnets_psc
+  project       = var.project_id
+  network       = local.network.name
+  region        = each.value.region
+  name          = each.value.name
+  ip_cidr_range = each.value.ip_cidr_range
+  purpose       = "PRIVATE_SERVICE_CONNECT"
+  description = lookup(
```

Comment:
Maybe mention in the description that this is a subnet for PSC NAT?

Reply:
done